analytics/analytics_test.go

```diff
@@ -0,0 +1,67 @@
+package analytics
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestAnalyticsSubmitPostsToHeap(t *testing.T) {
+	assert := assert.New(t)
+	var called bool
+	consentPath = newConsentTempfile(t)
+	ConsentGiven = true
+	WriteConsent()
+
+	// Setup
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		called = true
+
+		body, _ := ioutil.ReadAll(r.Body)
```

Comment:
Read the error and fail the test.

Reply:
Fixed in 9063fd03d10f0ea908f26bee701143a0a9ecc1ed.